reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java

```diff
@@ -61,18 +64,30 @@ public String getBestMatchVersionFor(GAV gav, List<String> availableVersions) {
         return findBiggestMatchingVersion(gav, availableVersions);
     }
 
+    private List<String> getBuiltVerionsFor0(List<String> allVersions) {
```

Comment:
Gotcha! Hopefully we won't have too many of these methods though! So I guess now just the typo in 'versions'